src/split.js

```diff
@@ -246,12 +258,14 @@ export default class Split extends React.Component {
             style={gutterStyle}
             gutterClassName={gutterClassName}
             onMouseDownAndTouchStart={this.handleMouseDownAndTouchStart}
+            gutter={gutter}
           />
         );
       }
 
       const style = {
         flexBasis: `${sizes[childNdx] * 100}%`,
+        width: `${sizes[childNdx] * 100}%`,
```

Comment:
Prevent the panel's relative positioning overriding the gutter position. Currently, the panel wraps its content and makes its content wrap-fit by nudging the gutter. This makes it so it doesn't nudge the gutter.